lib/util/exec.ts

```diff
@@ -4,13 +4,16 @@ import {
   ExecOptions as ChildProcessExecOptions,
 } from 'child_process';
 import { dockerCmd, DockerOptions } from './docker';
+import { getChildProcessEnv } from './env';
+import { defaultEnvVars } from './envVars';
```

Comment:
I tried so, but given up  because otherwise I ended up writing ugly boilerplate across lots of tests: 
```js
jest.mock('env.js', () => ({
  getChildProcessEnv: jest.fn(),
  defaultEnvVars: requireActual('env.js'),
})
```
instead of just `const env = mocked(_env);`

Reply:
Can we clear it up once we have completed the refactoring? I hope that env logic can be a mostly internal part of `exec` and not requiring multiple references.

Reply:
Ah, got it. Just realized now we have no one place where it's being called without subsequent `exec()`.

Reply:
So I think answer is yes, let's do it with the next PR maybe?